packages/sidebar/src/interfaces.ts

```diff
@@ -28,6 +28,12 @@ export interface ISidebarProps {
   maxWidth?: number;
   navWidth?: number;
   separatorWidth?: number;
+  rollWidth?: number;
+  onChangeWidth?: Function;
+  params?: {
```

Comment:
Props Params not needed. Collapsed and width must be on the same level as other props